### ISSUE TITLE
Win32EventLogAppender.dll Fails on Un*x environments

### ISSUE THREAD

**Issue by claudiow** [dblock/log4jna#31]
Some how a change was lost on the build.xml file fixing for next PR
```xml
    <target name="windmc" description="Update EventLogCategories.rc from .mc">
		<echo message="Building ${version}" />
		<condition property="mingw-prefix" value="">
			<os family="windows" />
		</condition>
		<property name="mingw-prefix" value="i586-mingw32msvc-" />
		<exec executable="windmc">
			<arg value="-h" />
			<arg file="${src.dir}" />
			<arg value="-r" />
			<arg file="${src.dir}" />
			<arg file="${src.dir}\EventLogCategories.mc" />
		</exec>
	</target>
```
Should be:
```xml
    <target name="windmc" description="Update EventLogCategories.rc from .mc">
		<echo message="Building ${version}" />
		<condition property="mingw-prefix" value="">
			<os family="windows" />
		</condition>
		<property name="mingw-prefix" value="i586-mingw32msvc-" />
		<exec executable="${mingw-prefix}windmc">
			<arg value="-h" />
			<arg file="${src.dir}" />
			<arg value="-r" />
			<arg file="${src.dir}" />
			<arg file="${src.dir}\EventLogCategories.mc" />
		</exec>
    </target>
```

**Comment by dblock** [comment]
PR this!

**Comment by claudiow** [comment]
Fixed on #33

**Comment by dblock** [comment]
Closed via #33.
